crates/engine/tree/src/tree/root.rs

```diff
@@ -509,28 +507,11 @@ fn update_sparse_trie(
                 )?;
             }
         }
-
-        storage_roots.insert(address, trie.storage_root(address).unwrap());
```

Comment:
let's leave `trie.storage_root(address).unwrap()` here since that's what we'll do in the future anyway